06-wrappers/02-UiButton/components/UiButton.vue

```diff
@@ -1,10 +1,43 @@
 <template>
-  <button class="button button_secondary button_block">BUTTON</button>
+  <component :is="tag" class="button" :class="{ [`button_${variant}`]: true, button_block: block }" v-bind="attrs"
+    ><slot
+  /></component>
```

Comment:
`v-bind="$attrs"` is redundant here. All the `$attrs` are inherited by default.

Note: `v-bind="$attrs"` would be required here in Vue 2. But in this case the component should also have `inheritAttrs: false`